auth/api/groups/http/transport.go

```diff
@@ -80,7 +75,38 @@ func DecodeListMemberGroupRequest(_ context.Context, r *http.Request) (interface
 		return nil, err
 	}
 
-	n, err := readStringQuery(r, nameKey)
+	m, err := readMetadataQuery(r, metadataKey)
+	if err != nil {
+		return nil, err
+	}
+
+	t, err := readBoolQuery(r, treeKey)
+	if err != nil {
+		return nil, err
+	}
+
+	req := listMembersReq{
+		token:    r.Header.Get("Authorization"),
+		id:       bone.GetValue(r, "groupID"),
```

Comment:
Create a const groupID

Reply:
@mteodor please do not mark comments as "resolved" without actually resolving them. Remove string constant.